.mdox.validate.yaml

```diff
@@ -1,4 +1,8 @@
 version: 1
+timeout: '1m'
+host_max_conns: 10
+cache:
+  type: 'SQLite'
```

Comment:
Maybe adding a `jitter` param with a duration of a few days would be good too? 🙂

Reply:
Oh, my bad. I thought there was a default jitter. Shall we use `validity/2` as jitter? What do you think?

Reply:
SGTM!

Reply:
Done in https://github.com/thanos-io/thanos/pull/5500/commits/7ec0faecec4827973a88db63ea31c549a25d8833